src/main/routes/govuk-entry-points.ts

```diff
@@ -0,0 +1,23 @@
+import * as express from 'express'
+import { Paths } from 'app/paths'
+import { Paths as FirstContactPaths } from 'first-contact/paths'
+import { ErrorHandling } from 'common/errorHandling'
+
+/* tslint:disable:no-default-export */
+// These routes are linked to from gov.uk and cannot be changed without a matching content change on their side
+export default express.Router()
+  .get(Paths.makeClaimReceiver.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      res.redirect('/not-implemented-yet')
+    })
+  )
+  .get(Paths.respondToClaimReceiver.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      res.redirect(FirstContactPaths.claimReferencePage.uri)
```

Comment:
We will need to rework that route to select non CMC reference numbers and take users to MCOL if non CMC number entered.

Reply:
in progress on ROC-3214